package.json

```diff
@@ -15,20 +51,7 @@
     "yo-yo": "^1.2.2"
   },
   "devDependencies": {
-    "electron-builder": "^5.10.5",
-    "electron-prebuilt": "^1.2.5",
+    "electron-prebuilt-compile": "1.6.11",
```

Comment:
This is what electron-forge uses to figure out what version of Electron to ship in-box